pkg/networkmanager/networkmanager.jsx

```diff
@@ -75,7 +75,7 @@ const App = () => {
                 <div id="networking-nm-crashed">
                     <EmptyStatePanel icon={ ExclamationCircleIcon }
                                      title={ _("NetworkManager is not running") }
-                                     action={ name ? _("Start service") : null }
+                                     action={_("Start service")}
```

Comment:
This was clearly meant to be `nmService.name` -- if it doesn't exist, we can't start it.